docs/developer/README.md

```diff
@@ -405,3 +405,11 @@ This function is triggered by the post_save event on the User model, designed to
 1. For New Users: Upon the creation of a new user, it checks for an existing `Contact` by email. If no matching contact is found, it creates a new Contact using the user's details from Login.gov. If a matching contact is found, it associates this contact with the user. In cases where multiple contacts with the same email exist, it logs a warning and associates the first contact found.
 
 2. For Existing Users: For users logging in subsequent times, the function ensures that any updates from Login.gov are applied to the associated User record. However, it does not alter any existing Contact records.
+
```

Comment:
If we are adding in switches, and now increasing dev tools to use both, please add documentation on when to use one vs the other, it can link to other documentation from the source if you want. If this needs to be a follow on ticket that's fine too but please make that ticket